Neos.Neos/Resources/Private/Fusion/Prototypes/NodeLink.fusion

```diff
@@ -0,0 +1,25 @@
+# Renders an <a> tag based on Neos.Neos:NodeUri pointing to a node
+#
+prototype(Neos.Neos:NodeLink) < prototype(Neos.Fusion:Tag) {
+  node = 'pass-the-node'
```

Comment:
Please do not define a string default value where a NodeInterface is required, i suggest to use `null` as default.

Reply:
Okay, I also found that weird. But in ImageUri it is also solved like that, which makes absolutely no sense if we expect an object. I'll change that.